src/layouts/Default.vue

```diff
@@ -30,15 +25,24 @@ export default {
 @import '../styles/variables.scss';
 @import '../styles/globals.scss';
 
+html {
+  font-size: 10px;
```

Comment:
Set up base font size so we can use relative units throughout the rest of the styles.